Resources/Prototypes/Entities/Clothing/Belt/job.yml

```
@@ -184,6 +184,12 @@
       - SurgeryTool
       - Dropper
       components:
+      # Carpmosia-start - civilian research tweaks
+      defib:
+        whitelist:
+          components:
+          - Defibrillator
+      # Carpmosia-end - civilian research tweaks
```

```suggestion
      - Defibrillator # Carpmosia-edit - civilian research tweaks
```
How did this get fucked up